### PR TITLE
fix: remove `aws-lambda` from expected npm version failures

### PR DESCRIPTION
**Pull request by G-Rath**
See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68667 - I think I'm doing this right...

### CODE REVIEW

**Comment by andrewbranch** [comment]
Yep, thank you!

File: .changeset/eleven-emus-warn.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@definitelytyped/dtslint": patch
+---
+
+remove `aws-lambda` from expected npm version failures
```
Comment (jakebailey) [inline .changeset/eleven-emus-warn.md:5]: Is this file pulled from the repo or from the package? If from the repo, we should update the workflow to not require this.

Comment (andrewbranch) [reply .changeset/eleven-emus-warn.md:5]: I think it's from the package right now. Would be nice to do it from the repo.

Comment (jakebailey) [reply .changeset/eleven-emus-warn.md:5]: Going through my email backlog from being out; I'll send a PR for this quick!

Comment (jakebailey) [reply .changeset/eleven-emus-warn.md:5]: Sent https://github.com/microsoft/DefinitelyTyped-tools/pull/959

**Pull request merged**